src/rgw/rgw_arrow.h

```diff
@@ -0,0 +1,15 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+#pragma once
+
+#include "include/common_fwd.h"
+
+namespace rgw::flight {
+
+    // initialize the Arrow Flight manager
```

Comment:
wrong indent.

Reply:
Do you mean that it's indented by 4 spaces rather than by 2? Or not by 0 spaces?

I've noticed that many like to indent namespaces by 0 and that may even be the convention in ceph code. However the instructions to emacs at the top of the source files make it indent one level in namespaces.

Reply:
sorry for the latency. we either use 2 spaces or 0 spaces for indent in a `namespace`. but i never found that we use 4 spaces in this case.